packages/vscode-extension/src/common/Project.ts

```diff
@@ -178,6 +178,8 @@ export interface ProjectInterface {
   startOrActivateSessionForDevice(deviceInfo: DeviceInfo): Promise<void>;
   terminateSession(deviceId: DeviceId): Promise<void>;
 
+  addToChatContext(...filePaths: string[]): void;
```

Comment:
NIT: I'd prefer if we passed the array here explicitly, rather than having a viariadic argument
```suggestion
  addToChatContext(filePaths: string[]): void;
```
The way it is now, it's not obvious when reading the code that calls this that the positions of the argumets don't matter.